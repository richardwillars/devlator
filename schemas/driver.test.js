const driverSchema = require("./driver");

describe("schemas/driver", () => {
  it("should export the schema", () => {
    expect(driverSchema).toMatchSnapshot();
  });
});
