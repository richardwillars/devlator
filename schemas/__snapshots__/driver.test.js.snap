// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`schemas/driver should export the schema 1`] = `
Object {
  "_id": Object {
    "required": true,
    "type": [Function],
    "unique": true,
  },
  "settings": Object {
    "default": Object {},
    "required": true,
    "type": [Function],
  },
}
`;
